composer.json

```diff
@@ -17,10 +17,11 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-config-aggregator": "^1.1",
         "laminas/laminas-stdlib": "^3.1",
-        "symfony/dependency-injection": "^3.4.36 || ^4.4.2 || ^5.0"
+        "psr/container": "1.1.1",
+        "symfony/dependency-injection": "^3.4.36 || ^4.4.2 || ^5.0 || ^6.0"
```

Comment:
please do not add symfony 6 if its not required to have support for PHP 8.1

Do we have any issues if we do not make this bump?

Reply:
Just to clarify. We do not actually verify if all these symfony versions are working in here as we do not have a matrix tto enforce this.
So pretty sure that only 3.4.36 & 5.4 (and something in between, but pretty sure not 4.x) are tested at all and if we now bump to 6.0+, we won't even test 5.4.

Reply:
IMO worth bumping to `^5.0 || ^6.0` anyway - if `^6.0` is not installed at all, then this diff is mostly fluff :\

Reply:
The whole thing is compatible with 7.2+ and we have a project where we need "symfony/dependency-injection" with 6.0. Therefore, I had increased this directly. Would that cause a problem?